src/System.Private.DataContractSerialization/src/System.Private.DataContractSerialization.csproj

```diff
@@ -13,7 +13,7 @@
     <PlatformTarget>AnyCPU</PlatformTarget>
     <DefineConstants Condition="'$(TargetGroup)'=='netcore50aot'">$(DefineConstants);NET_NATIVE</DefineConstants>
     <PackageTargetFramework Condition="'$(PackageTargetFramework)' == ''">netstandard1.3</PackageTargetFramework> 
-    
+    <PackageTargetRuntime Condition="'$(TargetGroup)' == 'netcore50aot'">aot</PackageTargetRuntime>
```

Comment:
Isn't this already set here: https://github.com/dotnet/corefx/blob/master/dir.props#L315